app/models/activity.rb

```diff
@@ -0,0 +1,7 @@
+class Activity < ApplicationRecord
+  validates :name, :amount, :categories, presence: true
+  validates :amount, numericality: { greater_than: 0 }
+
+  belongs_to :author, class_name: 'User'
+  has_and_belongs_to_many :categories
+end
```

Comment:
- [ ] Please validate the input fields for the `Category and transaction`. You have to flash or raise the message if a user tries to create the data without the information or with the empty data.